src/calculator.js

```diff
@@ -0,0 +1,37 @@
+export function add(num1, num2) {
+  if (num1 === undefined || num2 === undefined) {
+    throw new Error('Please give both values');
+  } else if (typeof num1 !== 'number' || typeof num2 !== 'number') {
+    throw new Error('Please add just numbers');
+  }
+  return num1 + num2;
+}
+
+export function substract(num1, num2) {
+  if (num1 === undefined || num2 === undefined) {
+    throw new Error('Please give both values');
+  } else if (typeof num1 !== 'number' || typeof num2 !== 'number') {
+    throw new Error('Please add just numbers');
+  }
+  return num1 - num2;
+}
+
+export function multiply(num1, num2) {
+  if (num1 === undefined || num2 === undefined) {
+    throw new Error('Please give both values');
+  } else if (typeof num1 !== 'number' || typeof num2 !== 'number') {
+    throw new Error('Please add just numbers');
+  }
+  return num1 * num2;
+}
+
+export function divide(num1, num2) {
+  if (num1 === undefined || num2 === undefined) {
+    throw new Error('Please give both values');
+  } else if (typeof num1 !== 'number' || typeof num2 !== 'number') {
+    throw new Error('Please add just numbers');
+  } else if (num2 === 0) {
+    throw new Error('Zero divition not possible');
+  }
+  return num1 / num2;
+}
```

Comment:
This ought to be a calculator object. In other words, there should be one exported object named calculator that returns these functions.
See this [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/testing_practice/calculator_requirement.png)